.github/workflows/rust.yml

```diff
@@ -100,11 +100,13 @@ jobs:
       - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
         with:
           rustflags: ''
-      - uses: taiki-e/install-action@nextest
+      - name: Install cargo-nextest
+        uses: taiki-e/install-action@nextest
       - name: "Test CoreCrypto xtask tooling"
         run: cargo nextest run --verbose --manifest-path xtask/Cargo.toml
       - name: "Test CoreCrypto benchmarks"
-        run: sh crypto/benches/scripts/test.sh
+        # Test only this because the others are executed in the benchmark workflow
+        run: cargo nextest run --bench mls_proteus
```

Comment:
Why not move this also to the benchmark workflow?

Reply:
Good point, will do